opencontest/contest/models/user.py

```diff
@@ -8,7 +8,9 @@
 
 class User:
     def __init__(self, username: str, fullname: str, password: str, type: str, id: str = None):
-        self.id = id
+        # Use uuid4 as an id because it is hard to brute
+        # force
+        self.id = id or str(uuid4())
```

Comment:
It is helpful when inspecting database files to have the username as part of the id. Maybe add it as a prefix, like this?
```
self.id = id or (username + str(uuid4()))
```